.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Pull request
+about: Create a pull request to contribute new features or bug fixes
+title: "[PULL REQUEST]"
+labels: pull_request
+
+---
+
+## Describe your changes and why you are making these changes
+
+## Related issue number (if any)
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code.
+- [ ] If this is a new feature, I have added unit tests and integration tests.
+- [ ] I have manually run the integration tests and they are passing.
+- [ ] If this is a UI-related feature, the UI still works after adding in the new feature.
```

Comment:
Tiny nit. I'd just change this to "All features on the UI continue to work correctly."

e.g., you could imagine someone changes some backend code that breaks the frontend because of an API/JSON field assumption.